consensus/polybft/contractsapi/init.go

```diff
@@ -14,306 +15,361 @@ const (
 
 var (
 	// Blade smart contracts
-	CheckpointManager               *artifact.Artifact
-	ExitHelper                      *artifact.Artifact
-	StateSender                     *artifact.Artifact
-	RootERC20Predicate              *artifact.Artifact
-	RootERC721Predicate             *artifact.Artifact
-	RootERC1155Predicate            *artifact.Artifact
-	ChildMintableERC20Predicate     *artifact.Artifact
-	ChildMintableERC721Predicate    *artifact.Artifact
-	ChildMintableERC1155Predicate   *artifact.Artifact
-	BLS                             *artifact.Artifact
-	BLS256                          *artifact.Artifact
-	System                          *artifact.Artifact
-	Merkle                          *artifact.Artifact
-	ChildValidatorSet               *artifact.Artifact
-	NativeERC20                     *artifact.Artifact
-	NativeERC20Mintable             *artifact.Artifact
-	StateReceiver                   *artifact.Artifact
-	ChildERC20                      *artifact.Artifact
-	ChildERC20Predicate             *artifact.Artifact
-	ChildERC20PredicateACL          *artifact.Artifact
-	RootMintableERC20Predicate      *artifact.Artifact
-	RootMintableERC20PredicateACL   *artifact.Artifact
-	ChildERC721                     *artifact.Artifact
-	ChildERC721Predicate            *artifact.Artifact
-	ChildERC721PredicateACL         *artifact.Artifact
-	RootMintableERC721Predicate     *artifact.Artifact
-	RootMintableERC721PredicateACL  *artifact.Artifact
-	ChildERC1155                    *artifact.Artifact
-	ChildERC1155Predicate           *artifact.Artifact
-	ChildERC1155PredicateACL        *artifact.Artifact
-	RootMintableERC1155Predicate    *artifact.Artifact
-	RootMintableERC1155PredicateACL *artifact.Artifact
-	L2StateSender                   *artifact.Artifact
-	CustomSupernetManager           *artifact.Artifact
-	StakeManager                    *artifact.Artifact
-	EpochManager                    *artifact.Artifact
-	RootERC721                      *artifact.Artifact
-	RootERC1155                     *artifact.Artifact
-	EIP1559Burn                     *artifact.Artifact
-	BladeManager                    *artifact.Artifact
-	GenesisProxy                    *artifact.Artifact
-	TransparentUpgradeableProxy     *artifact.Artifact
+	CheckpointManager               *contracts.Artifact
```

Comment:
We should get rid off all these `contracts.Artifact` fields and read them from the map. However, that can be a separate PR.